heaviside/activities.py

```diff
@@ -296,6 +319,22 @@ def fanout_nonblocking(args, session=None):
         handling_exception = False
         return args
 
+    except sfn.client.exceptions.ExecutionAlreadyExists:
+        # Don't kill running step functions if accidentally reuse name
+        handling_exception = False
+        # No need to report this exception to the step function's task
+        return args
```

Comment:
It seems like if you get into this handler you lose the sub_arg item that you popped off of the sub_args list.

Reply:
(same issue with the exception handler below)

Reply:
Don't think this is an issue since the handler below re-`raise`s.